safe/postprocessors/abstract_building_road_type_postprocessor.py

```diff
@@ -0,0 +1,236 @@
+# -*- coding: utf-8 -*-
+"""
+InaSAFE Disaster risk assessment tool developed by AusAid.
+
+Contact : ole.moller.nielsen@gmail.com
+
+.. note:: This program is free software; you can redistribute it and/or modify
+     it under the terms of the GNU General Public License as published by
+     the Free Software Foundation; either version 2 of the License, or
+     (at your option) any later version.
+"""
+
+__author__ = 'Marco Bernasocchi <marco@opengis.ch>'
```

Comment:
You should probably put in your own name when you create a new module. I know we have git blame but for someone reading the source, it is nice to know who write the module originally in case they want to contact them